android/app/src/main/java/app/organicmaps/util/Config.java

```diff
@@ -375,19 +376,37 @@ public static boolean isFirstLaunch(@NonNull Context context)
   public static void setFirstStartDialogSeen(@NonNull Context context)
   {
     MwmApplication.prefs(context)
-        .edit()
-        .putBoolean(KEY_MISC_FIRST_START_DIALOG_SEEN, true)
-        .apply();
+                  .edit()
+                  .putBoolean(KEY_MISC_FIRST_START_DIALOG_SEEN, true)
+                  .apply();
+  }
+
+  public static boolean isSearchPrivacyEnabled()
```

Comment:
Method name is confusing. If `isSearchPrivacyEnabled()` is true then history search is visible.
Let's rename it to `isSearchHistoryEnabled()`.
Also please rename `KEY_PREF_SEARCH_PRIVACY` to `KEY_PREF_SEARCH_HISTORY`.